pom.xml

```diff
@@ -321,11 +314,23 @@
         <groupId>io.grpc</groupId>
         <artifactId>grpc-netty</artifactId>
         <version>${grpc.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>io.netty</groupId>
+            <artifactId>*</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
what's the reason for adding this now?

Reply:
There is an explicit dependency on `netty-all`, this is redundant.

Reply:
I think it is better to specifically exclude the explicit dependencies that we rely on, rather than just `*`?

What is your opinion?